_includes/header.html

```diff
@@ -18,6 +18,7 @@
           <a
             class="navbar-item is-size-6 is-size-5-tablet is-hidden-mobile"
             href="{{ item.link }}"
+	    aria-label="{{ item.name }}"
```

Comment:
The content of this element already serves as the label, so we don't need to add a different label for screen reader users :)

```suggestion
```